### PR TITLE
template: Update bug template to include os version

### PR DESCRIPTION
**Pull request by mamoodi**
Creating a place for Operating System when opening a bug so it doesn't need to be asked.
Also makes figuring out which OS has the most issues.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_template.yml [hunk 1]
```diff
@@ -24,12 +24,20 @@ body:
       description: Provide a short description of the problem.
     validations:
       required: true
+      
+  - type: textarea
+    id: os-version
+    attributes:
+      label: Operating System
+      description: What Operating System are you using? Linux, Mac OS, WSL on Windows
```
Comment (mamoodi) [inline .github/ISSUE_TEMPLATE/bug_template.yml:32]: Is this a sufficient description?

Comment (li-boxuan) [reply .github/ISSUE_TEMPLATE/bug_template.yml:32]: looks good to me

File: .github/ISSUE_TEMPLATE/bug_template.yml [hunk 2]
```diff
@@ -24,12 +24,20 @@ body:
...
     validations:
       required: true
+      
+  - type: textarea
+    id: os-version
+    attributes:
+      label: Operating System
+      description: What Operating System are you using? Linux, Mac OS, WSL on Windows 
+    validations:
+      required: true
```
Comment (li-boxuan) [inline .github/ISSUE_TEMPLATE/bug_template.yml:34]: Can we make it non-required? Sometimes OS really doesn't matter. We already have a long list of mandatory fields that people have to fill in. Making the list longer will make people angrier 😆

Comment (mamoodi) [reply .github/ISSUE_TEMPLATE/bug_template.yml]: @li-boxuan removed the required haha

**Review by neubig — COMMENTED** [review]
Sorry about the repeated change requests, but because OS version is not required, could we move it below the required fields?

**Comment by mamoodi** [comment]
> Sorry about the repeated change requests, but because OS version is not required, could we move it below the required fields?

Of course!! Moved as per your recommendation.

**Review by li-boxuan — APPROVED** [review]
Thx!

**Pull request merged**